app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/BorrowedListAdapter.java

```diff
@@ -15,27 +15,33 @@
 import java.util.List;
 
 public class BorrowedListAdapter extends BaseBookListAdapter {
+    private BorrowedListAdapter.onItemClickListener itemClickListener;
 
-    public BorrowedListAdapter(List<Book> books) {
+    public interface onItemClickListener {
+        void run(int position);
+    }
+    public BorrowedListAdapter(List<Book> books,BorrowedListAdapter.onItemClickListener itemClickListener) {
```

Comment:
no need to specify `BorrowedListAdapter.onItemClickListener`, since they're in the same class, `OnItemClickListener` is fine.